### ISSUE TITLE
ENet returns a tensor with Nans as an output

### ISSUE THREAD

**Issue by kushnir95** [davidtvs/PyTorch-ENet#20]
I tried to train ENet from scratch on my own data and after several epochs of training ENet started to return a tensor with Nan values. Can any part of the model cause this problem? If it is true, how we can handle it?

**Comment by davidtvs** [comment]
Something like that is more likely to be connected to faulty input images (large values, NaN, Inf) or training parameters that cause instability (high learning rate).
